examples/python/find_coauthored_papers/README.md

```diff
@@ -0,0 +1,40 @@
+Given two author names, this script finds coauthored papers.
+In the case that multiple authors match either author name,
```

Comment:
Nit: this wording is a little confusing to me, I find something like "In the case there are multiple matches for either author's name" a bit more clear.